support-e2e/tests/test/adLiteCheckout.ts

```diff
@@ -0,0 +1,33 @@
+import test, { expect } from '@playwright/test';
+import { setupPage } from '../utils/page';
+import { completeCheckout } from './checkout';
+
+export type TestDetails = {
+	paymentType: string;
+	ratePlan: string;
+};
+
+export const testAdLiteCheckout = (testDetails: TestDetails) =>
```

Comment:
Is including the word `Checkout` in this function misleading, since we're testing more than just the checkout? We're testing the journey from the landing page. Would `testAdLitePurchase` be better? Or something else?